components/brave_wallet/browser/fil_tx_state_manager.h

```diff
@@ -29,9 +36,15 @@ class FilTxStateManager : public TxStateManager {
   FilTxStateManager(const FilTxStateManager&) = delete;
   FilTxStateManager operator=(const FilTxStateManager&) = delete;
 
+  std::unique_ptr<FilTxMeta> GetFilTx(const std::string& id);
+
  private:
   std::unique_ptr<TxMeta> ValueToTxMeta(const base::Value& value) override;
   std::string GetTxPrefPathPrefix() override;
+
+  std::string chain_id_;
+  std::string network_url_;
+  base::WeakPtrFactory<FilTxStateManager> weak_factory_;
```

Comment:
not needed?

Reply:
will be used later, but removed for now, thanks